### PR TITLE
Pair programmed with group on screen to finalize styling

### PR DESCRIPTION
**Pull request by kevncrypting**
- Changed color scheme
- Incorporated quiz page into locations page
- Finalized styling of site (including testimonial cards)
- Added functionality to submit button to clear previous info before populating new info

### CODE REVIEW

File: css/quiz.css
Comment (Giovannyf99) [inline css/quiz.css:1]: It is what it is.

File: location.html [hunk 1]
```diff
@@ -20,40 +21,77 @@ <h1>SuddenScapes</h1>
...
                 <a href="./quiz.html">
-                    <li><button class="btn btn-quiz">Take the Quiz</button></li>
+                    <li><button class="btn">Take the Quiz</button></li>
                 </a>
             </ul>
         </nav>
     </header>
     <!-- replaced div with ID of main-display with main as a more semantic way to reference the main section without having to use an ID -->
     <main>
+        <section class="quiz-container">
```
Comment (Giovannyf99) [inline location.html:31]: -very useful and informative quiz.

**Review by kblack-git — COMMENTED** [review]
MVP version to demo - great work putting together the final styling and logic!  Presentation time!

**Review by Giovannyf99 — COMMENTED** [review]
looks good.

**Pull request merged**